poppy/provider/akamai/background_jobs/check_cert_status_and_update/check_cert_status_and_update_tasks.py

```diff
@@ -28,6 +28,85 @@
 conf(project='poppy', prog='poppy', args=[])
 
 
+class PropertyGetHostnamesTask(task.Task):
+    default_provides = 'latest_version_hosts'
+
+    def __init__(self):
+        super(PropertyGetHostnamesTask, self).__init__()
+        service_controller, self.providers = \
+            memoized_controllers.task_controllers('poppy', 'providers')
+        self.akamai_driver = self.providers['akamai'].obj
+        self.existing_hosts = []
+
+    def execute(self, property_spec):
+        """Get the host names from the latest property"""
+        self.property_id = self.akamai_driver.papi_property_id(property_spec)
+
+        LOG.info('Starting to get latest version for property: %s'
+                 % self.property_id)
+        resp = self.akamai_driver.akamai_papi_api_client.get(
+            self.akamai_driver.akamai_papi_api_base_url.format(
+                middle_part='properties/%s' % self.property_id)
+        )
+        if resp.status_code != 200:
+            raise RuntimeError('PAPI API request failed.'
+                               'Exception: %s' % resp.text)
+        else:
+            a_property = json.loads(resp.text)['properties']['items'][0]
+            latestVersion = a_property['latestVersion'] or 0
```

Comment:
[Here is a streamlined](https://developer.akamai.com/api/core_features/property_manager/v1.html#getlatestversion) way of grabbing the latest version.